var/spack/repos/builtin/packages/py-pkgconfig/package.py

```diff
@@ -20,7 +20,7 @@ class PyPkgconfig(PythonPackage):
     depends_on("python@3.3:", when="@1.5.5:", type=("build", "run"))
     depends_on("python@2.7,3.3:", when="@1.5:", type=("build", "run"))
     depends_on("python@2.6:2.7,3.3:", when="@1.4:", type=("build", "run"))
-    depends_on("py-poetry-core@1:", when="@1.5.5:", type="build")
+    depends_on("py-poetry-core@1:", when="@1.5.2:", type="build")
     depends_on("py-poetry-core@0.12:", when="@1.5:", type="build")
```

Comment:
Just took another look at this and it looks like it should actually be:
```suggestion
    depends_on("py-poetry-core@1:", when="@1.5.2:", type="build")
    depends_on("py-poetry@0.12:", when="@1.5.0:1.5.1", type="build")
```